internal/oapiffi/ffi2swagger.go

```diff
@@ -42,7 +42,22 @@ func NewFFISwaggerGen() FFISwaggerGen {
 func (og *ffiSwaggerGen) Generate(ctx context.Context, baseURL string, api *fftypes.ContractAPI, ffi *fftypes.FFI) (swagger *openapi3.T) {
 	hasLocation := !api.Location.IsNil()
 
-	routes := []*oapispec.Route{}
+	routes := []*oapispec.Route{
+		{
+			Name:            "interface",
+			Path:            "interface", // must match a route defined in apiserver routes!
+			Method:          http.MethodGet,
+			JSONInputValue:  nil,
+			JSONOutputValue: func() interface{} { return &fftypes.FFI{} },
+			JSONOutputCodes: []int{http.StatusOK},
+		},
+	}
+	for _, method := range ffi.Methods {
+		routes = og.addMethodMetadata(routes, method)
+	}
+	for _, event := range ffi.Events {
+		routes = og.addEventMetadata(routes, event)
+	}
```

Comment:
After coming back to consider these along with the additional routes added in #696, I think these extra routes under `/interface/methods` and `/interface/events` are not necessary. The `/interface` helper would actually be the only one I vote to keep from this PR now.

Reply:
Pushed another commit to remove these routes completely.